sirepo/package_data/static/js/raydata.js

```diff
@@ -741,6 +746,16 @@ SIREPO.app.directive('scanSelector', function() {
                 $scope.selectAllColumns = ! $scope.selectAllColumns;
             };
 
+            $scope.unselectAllScans = () => {
+                Object.keys(appState.models.selectedScans.uids).forEach((u) => {
```

Comment:
I think we can just say `appState.models.selectedScans.uids = []` instead of deleting each one individually.